libs/dkg/include/dkg/dkg_service.hpp

```diff
@@ -148,7 +148,9 @@ class DkgService : public ledger::EntropyGeneratorInterface
   }
   std::string GroupPublicKey()
   {
-    return dkg_.public_key();
+    // TODO(HUT): chat to Jenny
+    //return dkg_.PublicKey();
```

Comment:
note

Reply:
This will go away. I have put it here only to for checking the DKG is working correctly.